src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -187,12 +193,21 @@ Activate the `PaymentRequest` interface by calling its [`show`](https://www.w3.o
 *PaymentRequest show method*
 
 ### Abort a Payment Request {: #abort-paymentrequest }
-You can intentionally abort a `PaymentRequest` by calling its [`abort`](https://www.w3.org/TR/payment-request/#abort) method. Use this method if the app needs to cancel the payment request after the `show` method has been called but before the promise has been resolved&mdash;for example, if an item is no longer available, or the user fails to confirm the purchase within an allotted amount of time.
+You can intentionally abort a `PaymentRequest` by calling its [`abort`](https://www.w3.org/TR/payment-request/#abort) method. This is particulary useful when the shopping session is timed out or the goods is sold out in between a user's transaction.
```

Comment:
Change 'the goods is sold out in between a user's transaction' to 'an item in the cart sells out during the transaction'.